server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -197,7 +197,7 @@ public void run() {
         }
 
         if (currentTServers.isEmpty()) {
-          eventListener.waitForEvents(Manager.TIME_TO_WAIT_BETWEEN_SCANS);
+          eventListener.waitForEvents(manager.getWaitTimeBetweenScans());
```

Comment:
Using the Supplier idea I suggested, the value can be obtained once at the top of this loop, and reused for the rest of the loop iteration, using a final local variable, so it doesn't change in the middle of the iteration.

Reply:
baea8d7 provides the TabletGroupWatcher with the current value of the property.